### PR TITLE
Add utility zone stats

### PR DESCRIPTION
**Pull request by mem48**
Zone stats for utility trips

### CODE REVIEW

File: R/zone_stats.R [hunk 1]
```diff
@@ -235,6 +235,92 @@ export_zone_json <- function(x,  idcol = "DataZone", path = "outputdata/json", z
...
+                                orig_public_transport_ebike = sum(public_transport_ebike, na.rm = TRUE),
+                                orig_foot_ebike = sum(foot_ebike, na.rm = TRUE),
+                                orig_taxi_ebike = sum(taxi_ebike, na.rm = TRUE),
+                                
+                                orig_quietness = mean(quietness, na.rm = TRUE),
+                                orig_route_hilliness = mean(route_hilliness, na.rm = TRUE)
+  )
+  
+  
+  comm_to <- dplyr::group_by(comm, endDZ, purpose)
```
Comment (Robinlovelace) [inline R/zone_stats.R:279]: Consistent assignment operator please.

**Review by Robinlovelace — CHANGES_REQUESTED** [review]
Looks good, minor tweak to code style for consistency requested.

File: _targets.R [hunk 2]
```diff
@@ -642,9 +645,11 @@ tar_target(school_stats_from_baseline, {
                      foot = sum(foot, na.rm = TRUE),
                      public_transport = sum(public_transport, na.rm = TRUE),
                      other = sum(other, na.rm = TRUE))
+  stats$schooltype = paste0("schl_",stats$schooltype)
   stats = tidyr::pivot_wider(stats, 
                              id_cols = c("DataZone"),
                              names_from = c("schooltype"),
+                             names_glue = "{schooltype}_{.value}",
```
Comment (Robinlovelace) [inline _targets.R:652]: Nice syntax.

**Review by Robinlovelace — COMMENTED** [review]
Let me give it a spin to ensure it works. Will approve + merge ASAP to keep the momentum up.

**Comment by Robinlovelace** [comment]
Heads-up @mem48 I'm planning to push to this branch, let me know if you're making any other additional changes.

**Comment by Robinlovelace** [comment]
@mem48 I get this error:

```
▶ start target save_outputs
✖ error target save_outputs
▶ end pipeline [1.764 minutes]
Warning messages:
1: Thats a lot of columns for the GeoJSON, are they all needed? 
2: Thats a lot of columns for the GeoJSON, are they all needed? 
3: Thats a lot of columns for the GeoJSON, are they all needed? 
4: Thats a lot of columns for the GeoJSON, are they all needed? 
5: Thats a lot of columns for the GeoJSON, are they all needed? 
6: running command 'cd outputdata; tippecanoe -o data_zones.pmtiles --name=data_zones --layer=data_zones --attribution=UniverstyofLeeds --minimum-zoom=6 -zg --coalesce-smallest-as-needed --detect-shared-borders --extend-zooms-if-still-dropping --maximum-tile-bytes=5000000 --simplification=10 --buffer=5 --force  data_zones.geojson' had status 110 
7: running command 'cd outputdata; tippecanoe -o dasymetric_verylow.pmtiles --name=dasymetric --layer=dasymetric --attribution=OS --minimum-zoom=4 --maximum-zoom=6 --coalesce-smallest-as-needed --detect-shared-borders --maximum-tile-bytes=5000000 --simplification=1 --buffer=5 --force dasymetric_verylow.geojson; tippecanoe -o dasymetric_low.pmtiles --name=dasymetric --layer=dasymetric --attribution=OS --minimum-zoom=7 --maximum-zoom=9 --coalesce-smallest-as-needed --detect-shared-borders --maximum-tile-bytes=5000000 --simplification=1 --buffer=5 --force dasymetric_low.geojson; tippecanoe -o dasymetric_med.pmtiles --name=dasymetric --layer=dasymetric --attribution=OS --minimum-zoom=10 --maximum-zoom=14 --coalesce-smallest-as-needed --detect-shared-borders --maximum-tile-bytes=5000000 --simplification=2 --buffer=5 --force dasymetric_med.geojson; tippecanoe -o dasymetric_high.pmtiles --name=dasymetric --layer=dasymetric --attribution=OS -zg --minimum-zoom=15 --extend-zooms-if-still-dropping - [... truncated] 
8: 4 targets produced warnings. Run targets::tar_meta(fields = warnings, complete_only = TRUE) for the messages. 
Error:
! Error running targets::tar_make_future()
  Error messages: targets::tar_meta(fields = error, complete_only = TRUE)
  Debugging guide: https://books.ropensci.org/targets/debugging.html
  How to ask for help: https://books.ropensci.org/targets/help.html
  Last error: Some files do not exist
```

**Comment by Robinlovelace** [comment]
From the terminal:

```
cd outputdata; tippecanoe -o data_zones.pmtiles --name=data_zones --layer=data_zones --attribution=UniverstyofLeeds --minimum-zoom=6 -zg --coalesce-smallest-as-needed --detect-shared-borders --extend-zooms-if-still-dropping --maximum-tile-bytes=5000000 --simplification=10 --buffer=5 --force  data_zones.geojson
data_zones.geojson: No such file or directory
0 features, 4 bytes of geometry, 0 bytes of string pool
Did not read any valid geometries
```

**Review by atumscott — APPROVED** [review]

**Review by Robinlovelace — APPROVED** [review]

**Pull request merged**